genon/preprocessor/facade/attachment_processor.py

```
@@ -457,6 +457,140 @@ def install_packages(packages):
+            timeout=timeout,
+        )
+        resp.raise_for_status()
+        body = resp.json()
+        if body.get("code") != 0:
+            raise RuntimeError(f"workflow code={body.get('code')} {body.get('errMsg')}")
+        data = body.get("data") or {}
+        infos = data.get("sensitive_infos")
+        if infos is None:  # text 필드에 JSON 문자열로 실려온 경우 파싱 시도
+            try:
+                infos = (json.loads(data.get("text") or "{}")).get("sensitive_infos")
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)